experimental/GModule/src/Cohomology.jl

```diff
@@ -463,7 +466,10 @@ import Hecke.⊗
 ⊗(C::GModule...) = Oscar.tensor_product(C...; task = :none)
 
 
-function Oscar.tensor_product(F::AbstractAlgebra.FPModule{T}...; task = :none) where {T}
+function Oscar.tensor_product(F::AbstractAlgebra.FPModule{T}, Fs::AbstractAlgebra.FPModule{T}...; task = :none) where {T}
+  return Oscar.tensor_product([F, Fs...]; task)
+end
+function Oscar.tensor_product(F::Vector{AbstractAlgebra.FPModule{T}}; task = :none) where {T}
```

Comment:
```suggestion
function Oscar.tensor_product(F::Vector{<: AbstractAlgebra.FPModule{T}}; task = :none) where {T}
```
Or what was the Aqua-friendly way to do this?

Reply:
For vectors your suggestion is Aqua-friendly. Only for varargs, it sometimes complains